cloud_function/database-migration/migrate_database.py

```
@@ -390,6 +395,67 @@ def create_table_structure(source_conn, dest_conn, table_name):
+        for col_name, col_info in source_columns.items():
+            if col_name not in dest_columns:
+                data_type, char_len, num_prec, num_scale, is_null, col_default = col_info
+                col_def = get_column_definition(data_type, char_len, num_prec, num_scale, is_null, col_default)
+                
+                try:
+                    alter_stmt = f'ALTER TABLE "{table_name}" ADD COLUMN "{col_name}" {col_def};'
+                    dest_cursor.execute(alter_stmt)
+                    dest_conn.commit()
+                    log(f"    [OK] Added column: {table_name}.{col_name}")
+                    changes_made = True
+                except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)